adapters/yieldlab/usersync.go

```diff
@@ -0,0 +1,12 @@
+package yieldlab
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewYieldlabSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("yieldlab", 0, temp, adapters.SyncTypeRedirect)
```

Comment:
It looks like Yieldlab is registered in the GVL with vendor id 70. Is that correct?

Reply:
Yes, that is correct. When we created the adapter, we were in the process of registering for TCFv2.0 and didn't know, if vendors are getting the same id.

```suggestion
	return adapters.NewSyncer("yieldlab", 70, temp, adapters.SyncTypeRedirect)
```